src/Microsoft.AspNet.Authentication.JwtBearer/JwtBearerHandler.cs

```diff
@@ -37,7 +37,7 @@ internal class JwtBearerHandler : AuthenticationHandler<JwtBearerOptions>
 
                 if (receivingTokenContext.Skipped)
                 {
-                    return AuthenticateResult.Success(ticket: null);
+                    return AuthenticateResult.Failed("Skipped.");
```

Comment:
Failed doesn't actually mean anything. It's just syntax sugar to make the code easier to read. It just allows a null ticket :). We can rename the static helpers to align with this intent.